### ISSUE TITLE
Does this work on an app running on multiple instances?

### ISSUE THREAD

**Issue by elie222** [mizzao/meteor-user-status#70]
I'm using meteorhacks:cluster and wondering whether online will work if a user is connected to one instance, and not to another.

**Comment by mizzao** [comment]
No, it doesn't - but we started a discussion of this a while ago in #21.

**Comment by elie222** [comment]
Any idea if https://atmospherejs.com/konecty/multiple-instances-status does
a good job?

**Comment by elie222** [comment]
It did work well for me if anyone else is wondering.

**Comment by derwaldgeist** [comment]
+1 for multi-instance support

**Comment by Goatic** [comment]
:+1:  for multi-instance support

**Comment by Alveoli** [comment]
+1 for multi-instance support!
